app/buy/index.js

```diff
@@ -42,6 +42,7 @@ const Buy = () => {
         <Page header="Select Location" style={styles.page}>
             <ScrollView contentContainerStyle={styles.scrollContainer}>
                 {options.map((option) => (
+                    // <View style={styles.location} key={option.label}>
```

Comment:
Can we remove these comments or is there a reason they're still here?

Reply:
smh